mitodl/ocw-studio
# Make starter required for OCW_IMPORT_STARTER_SLUG

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #514 

#### What's this PR do?
 - Remove a constant which was also being used as the course starter slug. Instead testing code will use the setting
 - If the starter does not exist on import, an exception will be thrown

We also need to make sure our production environments have environment variables which match the starter slugs

#### How should this be manually tested?
Run `./manage.py import_ocw_course_sites -b ocw-to-hugo-output-qa --filter res-8-001-applied-geometric-algebra-spring-2009`. The website should have the starter populated correctly, or else there should be an error in the import or when Django starts and finds a missing environment variable.
